xarray/backends/api.py

```diff
@@ -233,6 +233,30 @@ def maybe_decode_store(store, lock=False):
     return maybe_decode_store(store)
 
 
+def open_dataarray(*args, **kwargs):
+    """
+    Opens an `xarray.DataArray` from a netCDF file, and returns the single
+    variable that is within the file. If multiple variables are present then
+    a ValueError is raised.
+
+    This is designed to read files saved with `xarray.DataArray.to_netcdf`.
+
+    Parameters
+    ----------
+    All parameters are passed directly to `xarray.open_dataset`.
+    """
+    dataset = open_dataset(*args, **kwargs)
```

Comment:
One downside of this approach is that it leaves a stranded file open without any possible way to close it.

We might want to add a `.close()` method to DataArray so we can clean this up. Or maybe just save that for a later PR.